nautilus/CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ set_target_properties(nautilus PROPERTIES
         EXPORT_NAME nautilus
 )
 
+disable_target_warnings(fmt)
```

Comment:
should this be here? it causes an cmake error if external fmt is used

Reply:
I moved this. It should work now.